waiter/test/waiter/mesos/utils_test.clj

```diff
@@ -0,0 +1,64 @@
+;;
+;;       Copyright (c) 2017 Two Sigma Investments, LP.
+;;       All Rights Reserved
+;;
+;;       THIS IS UNPUBLISHED PROPRIETARY SOURCE CODE OF
+;;       Two Sigma Investments, LP.
+;;
+;;       The copyright notice above does not evidence any
+;;       actual or intended publication of such source code.
+;;
+(ns waiter.mesos.utils-test
+  (:require [clojure.core.async :as async]
+            [clojure.data.json :as json]
+            [clojure.test :refer :all]
+            [qbits.jet.client.http :as http]
+            [waiter.mesos.utils :refer :all])
+  (:import (clojure.lang ExceptionInfo)))
```

Comment:
Nitpick: this can just be `clojure.lang.ExceptionInfo`

Reply:
Done.